include/albatross/src/models/sparse_gp.hpp

```diff
@@ -16,9 +16,13 @@
 namespace albatross {
 
 namespace details {
+
 constexpr double DEFAULT_NUGGET = 1e-12;
-const std::string measurement_nugget_name = "measurement_nugget";
-const std::string inducing_nugget_name = "inducing_nugget";
+
+inline std::string measurement_nugget_name() { return "measurement_nugget"; }
```

Comment:
Potential small optimization for later would be to declare the string as a `static const` variable inside the function and return a `const &` to it, to avoid calling the `std::string` constructor each time the function is called. No need for that now, or probably ever since it's such a small optimization.